tests/suites/test_suite_aes.xex.data

```diff
@@ -0,0 +1,3599 @@
+AES-128-XEX Encrypt NIST XTSTestVectors #1
```

Comment:
Please add a few negative tests: lengths that are not a multiple of 16; length=0; AES-192.